day03/ex00/ex00.md

```diff
@@ -0,0 +1,143 @@
+
+We can achieve high availability by deploying your application across multiple availability zones.
+
+![AWS regions](../assets/aws_regions.png){width=400px}
+
+The `edge locations` you see on the picture are endpoints for AWS which are used for caching content (performance optimization mechanism in which data is delivered from the closest servers for optimal application performance). Typically consists of CloudFront (Amazon's content delivery network (CDN)).
+
+## AWS account registration
+
+When we create a new account on AWS, we have an AWS Free Tier. It allows us to use some of AWS resources for free each month during one year.
+
+In order to create an AWS account, we must go to the following [address](https://aws.amazon.com).
+
+Then we click on the create an account button and fill the following form.
+
+![Account creation 1](../assets/account_creation_1.png){width=240px}
+
+\newpage
+
+![Account creation 2](../assets/account_creation_2.png){width=245px}
+
+Don't worry! Even if we enter our card number, this day is not going to cost us anything. First,  Indeed, AWS as a free tier usage which allows us to use a small amount of AWS resources for free. This will be sufficient for what we want to do during this day. At the end of your day you will need to destroy your infrastructure entirely (don't keep things runing) !!!
+
+In the next form, we choose the `basic plan`, which is free.
+
+To connect to aws we use our email and password. We will then access to your root account. The root account **MUST NOT** be used directly when using AWS resources for security purposes. We will later create IAM users to use AWS resources more safely.
+
+
+## Discover AWS console
+
+The console allows to search for specific services. by default they are sorted by group but can be sorted alphabetically.
+
+![AWS console](../assets/console.png){width=600px}
+
+We notice there is a link for resource group. Resource groups allow us to take a collection of aws resources and assign a tag (a label) to them so we can manage them as a group.
+
+The pushpin is for one-click navigation. It allows us to create shortcuts for most commonly used resources.
+
+![AWS pushpin](../assets/pushpin.png){width=600px}
+
+The alarm icon allows us to see all system alerts. If we click we will see more details about the issues and their current status.
+
+![AWS alerts](../assets/alerts.png){width=200px}
+
+Then we have account information and the selected AWS regions we are working on (all resources are not available for every regions).
+
+Support center allows us to create cases when we encounter problems (we can see support plan here, which will influence aws response time).
+
+
+## Setting up a billing alarm
+
+In order to avoid any big mistake (forget to destroy your infrastructure and it cost you some money), we are going to set an alarm to tell us if we have a cost superior to 1$.
```

Comment:
```suggestion
In order to avoid any big mistake (like forgetting to destroy your infrastructure and facing unexpected expenses), you are going to set an alarm for cost overrun. You will receive alerts when charges have exceeded 1$.
```